target/linux/ar71xx/base-files/etc/board.d/01_leds

```diff
@@ -138,6 +138,12 @@ dragino2)
 	ucidef_set_led_netdev "lan" "LAN" "$board:red:lan" "eth0"
 	ucidef_set_led_netdev "wan" "WAN" "$board:red:wan" "eth1"
 	;;
+dw33d)
+	ucidef_set_led_usbdev "mmc" "MMC" "dw33d:blue:mmc" "1-1"
+	ucidef_set_led_usbdev "usb" "USB" "dw33d:blue:usb" "2-1"
+	ucidef_set_led_netdev "internet" "INTERNET" "dw33d:blue:internet" "eth0" "rx"
+	ucidef_set_led_wlan "wlan2g" "WLAN-2.4G" "dw33d:blue:wlan-2g" "phy1tpt"
+	;;
```

Comment:
Sorry for nagging again. Please use $board here as well.

You can drop the "rx" param from ucidef_set_led_netdev. In that case the LED should react to rx, tx and link.